ci-operator/config/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master.yaml

```diff
@@ -0,0 +1,37 @@
+canonical_go_repository: github.com/redhat-developer/devconsole-operator
+
+resources:
+  '*':
+    limits:
+      memory: 1Gi
+    requests:
+      memory: 250Mi
+
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/build-root/Dockerfile
```

Comment:
This is the reason why the `unit` rehearsal fails, this file does not seem to exist in your repo (https://github.com/redhat-developer/devconsole-operator). If you intend to add it there, it is fine to merge this PR as it is, rehearsals are not blocking.

Reply:
Thanks! I've added the right path.